### DISCUSSION TITLE
Could I create node.js port of ripgrep?

### DISCUSSION THREAD

**Discussion by jqhr** [BurntSushi/ripgrep#2065] [General] [answered]
I want write an node.js project, it need grep or other tools like it. I cannot find some grep package on npmjs.org. So I want to create a node.js port of ripgrep. 
I want to use wasm.

**Comment by BurntSushi** [comment, marked as answer]
You could? Sure. Anything is possible. Sorry for not being helpful, but I'm not sure what else you want me to say here. Your question is really too broad and about topics which I know little about (Node and wasm).

**Reply by jqhr** [reply]
Good!
I want use some code Iike this in js.
```js
const ripgrep = require('ripgrep')
const result = ripgrep.search('some text'',  'some path')
console.log(result)
```

**Reply by BurntSushi** [reply]
Go for it!
